### PR TITLE
Drop references to python 2.6 and lower in docs

### PR DESCRIPTION
**Pull request by beadysea**
In view of #261, this updates installation instructions and readme.

### CODE REVIEW

File: docs/user/install.rst [hunk 1]
```diff
@@ -55,7 +68,7 @@ For Python 3.3 or 3.4
 - Microsoft Visual C++ 10.0 standalone: Windows SDK 7.1 (x86, x64, ia64)
 - Microsoft Visual C++ 10.0 with Visual Studio 2010 (x86, x64, ia64)
 
-For Python 2.6, 2.7, 3.0, 3.1, 3.2
```
Comment (rgov) [inline docs/user/install.rst:58]: I'm not a Windows guy, can we change this to `3.5+`? I don't know if we want to support these old versions of Python 3.

File: docs/user/install.rst [hunk 2]
```diff
@@ -13,6 +13,15 @@ with Python 2 and 3.
 
 	pip install pybluez
 
+(there are also binaries for Windows platform on PyPI or here - `Unofficial Windows Binaries for Python Extension Packages <https://www.lfd.uci.edu/~gohlke/pythonlibs/#pybluez>`)
+
+For experimental Bluetooth Low Energy support(only for Linux platform -
```
Comment (hugovk) [inline docs/user/install.rst]:
```suggestion
For experimental Bluetooth Low Energy support (only for Linux platform -
```

File: docs/user/install.rst [hunk 3]
```diff
@@ -13,6 +13,15 @@ with Python 2 and 3.
 
 	pip install pybluez
 
+(there are also binaries for Windows platform on PyPI or here - `Unofficial Windows Binaries for Python Extension Packages <https://www.lfd.uci.edu/~gohlke/pythonlibs/#pybluez>`)
+
+For experimental Bluetooth Low Energy support(only for Linux platform -
+for additional dependencies please take look at:
+`ble-dependencies <https://bitbucket.org/OscarAcena/pygattlib/src/45e04060881a20189412681f52d55ff5add9f388/DEPENDS?at=default>`
```
Comment (hugovk) [inline docs/user/install.rst]:
```suggestion
`ble-dependencies <https://bitbucket.org/OscarAcena/pygattlib/src/45e04060881a20189412681f52d55ff5add9f388/DEPENDS?at=default>`)
```

**Review by hugovk — COMMENTED** [review]
Some minor nits

File: docs/user/install.rst (hunk 1 above)
Comment (beadysea) [reply docs/user/install.rst:58]: Should we also remove Windows XP?  There are people out there still running on it but I think it's time to consider removing support for it considering Windows 7 reaches end of support in 2020.

File: docs/user/install.rst (hunk 2 above)
Comment (beadysea) [reply docs/user/install.rst]: Agreed.

File: docs/user/install.rst (hunk 3 above)
Comment (beadysea) [reply docs/user/install.rst]: Thanks @hugovk. I'll batch your suggestions.

File: docs/user/install.rst (hunk 1 above)
Comment (rgov) [reply docs/user/install.rst:58]: Yes I think if you are running a version of Windows from 2001 you can figure out how to install an old version of PyBluez also.

File: README.md [hunk 4]
```diff
@@ -91,50 +91,7 @@ active.
...
--   Python 2.3 or more recent version
-
-#### macOS
-
--   Python 2.3 or later
--   Xcode
--   PyObjc 3.1b or later
-    (<https://pythonhosted.org/pyobjc/install.html#manual-installation>)
-
+Please refer to the [PyBluez Installation page](https://pybluez.readthedocs.io/en/latest/user/install.html) on our [Read the Docs site.](https://pybluez.readthedocs.io/)
```
Comment (rgov) [inline README.md]: I think this is creating a cyclical reference. The way I see it, Read the Docs is a nice hosted version of the documentation in the repo, but the real "source of truth" should be inside the repository.

For instance, if these install instructions are different for a specific branch, or if they are looking at a historical checkout of the source, pointing to the latest docs on RtD is not necessarily correct.

```markdown
Please refer to the [installation instructions](/docs/user/install.rst).
```

What do you think?

**Review by rgov — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (beadysea) [reply README.md]: Yes I see your point. While updating install.rst should auto-update RtD, pointing to the internal file would save updating readme on every release.

Comment (beadysea) [reply README.md]: Updated.

File: docs/user/install.rst (hunk 1 above)
Comment (beadysea) [reply docs/user/install.rst:58]: Removed XP, Vista and references to python below 3.5 for windows.

**Comment by beadysea** [comment]
~~Might need to merge #253 before this one. Both change install.rst and this PR supersedes the earlier changes~~.

File: docs/install.rst [hunk 5]
```diff
@@ -1,79 +1,85 @@
...
+- Widcomm BTW development kit 5.0 or later (Optional)
+
+macOS Dependencies
+"""""""""""""""""" 
+- Xcode
+- PyObjc 3.1b or later (https://pythonhosted.org/pyobjc/install.html#manual-installation)
+
+
+
+.. _master.zip: https://github.com/pybluez/pybluez/archive/master.zip
```
Comment (rgov) [inline docs/install.rst]: Can you please move these up closer to where they are used? Right below the table is fine. I like things to stay close to where they are used.

File: README.md [hunk 6]
```diff
@@ -91,50 +91,7 @@ active.
...
--   Python 2.3 or more recent version
-
-#### macOS
-
--   Python 2.3 or later
--   Xcode
--   PyObjc 3.1b or later
-    (<https://pythonhosted.org/pyobjc/install.html#manual-installation>)
-
+Please refer to the [installation instructions](/docs/user/install.rst).
```
Comment (rgov) [inline README.md]: As part of this pull request, you renamed the file to remove the `user/` part.

**Review by rgov — CHANGES_REQUESTED** [review]

**Review by rgov — APPROVED** [review]

**Pull request merged**